tensorboard/plugins/scalar/tf_scalar_dashboard/tf-scalar-card.ts

```diff
@@ -270,14 +282,19 @@ export class TfScalarCard extends PolymerElement {
     onLoad,
     onFinish
   ) => {
-    const inColab = initialURLSearchParams.get('tensorboardColab') === 'true';
-    if (inColab) {
-      // Google-internal Colab doesn't support HTTP POST requests, so we fall
-      // back to HTTP GET (even though public Colab supports POST).
-      return this._requestDataGet(items, onLoad, onFinish);
-    } else {
-      return this._requestDataPost(items, onLoad, onFinish);
-    }
+    // TODO(psybuzz): Ideally, we would always make use of `_requestDataPost`,
```

Comment:
Could we get away with a smaller delta here and leave most of the code intact so it's easy to revert?  E.g.

```typescript
// TODO(b/170675710): Stop unconditionally forcing legacy endpoint when fixed.
const forceLegacyEnpoint = true;
if (inColab || forceLegacyEndpoint) {
  return this._requestDataGet(items, onLoad, onFinish);
} else {
  return this._requestDataPost(items, onLoad, onFinish);
}
```

That makes it obvious that we'll still need to the `inColab` check after resolving the TODO, which is better IMO than stating this fact in a comment.  And I'm not sure we need to link to the spanner PR given that the gating factor isn't anything to do with the PR per se but rather with Google-internal release details which aren't particularly visible or meaningful outside Google anyway.

Reply:
That's much more concise!  Done.